README.md

```diff
@@ -42,21 +39,36 @@ You need to classify also the psick class from the psick module (which by defaul
 specific Hiera data it does nothing) in order to leverage on the general variables
 set in the main psick class and used in the psick modules.
 
+In short in you manifests/site.pp or wherever you classify your nodes you need to:
+
+    include psick
+
+and then include/classify the psick profiles you want, for example:
+
+    include psick_profile::gitlab
+    include psick_profile::grafana
+
+
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/example42/puppet-psick_profile/pullRequest?prid=9480555)
```suggestion

```